pandas/tests/tseries/offsets/test_offsets_properties.py

```diff
@@ -36,8 +36,8 @@
 
 with warnings.catch_warnings():
     warnings.simplefilter("ignore")
-    min_dt = (pd.Timestamp(1900, 1, 1).to_pydatetime(),)
-    max_dt = (pd.Timestamp(1900, 1, 1).to_pydatetime(),)
+    min_dt = pd.Timestamp(1900, 1, 1).to_pydatetime()
```

Comment:
I *think* this might be intentional since these are arguments to hypothesis and may want to parametrize, though @TomAugspurger might have more insight.

@aaditya-panik can you advise the error this was causing?

Reply:
I'm not sure why this was a tuple before.

Reply:
Sounds good thanks for confirming

Reply:
> I _think_ this might be intentional since these are arguments to hypothesis and may want to parametrize, though @TomAugspurger might have more insight.
> 
> @aaditya-panik can you advise the error this was causing?

The error was caused because hypothesis.datetimes expects a Datetime object as argument rather than a tuple.